cluster/addons/fluentd-gcp/fluentd-gcp-ds.yaml

```diff
@@ -42,10 +42,10 @@ spec:
           value: --no-supervisor -q
         resources:
           limits:
-            memory: 300Mi
+            memory: 300Mi # fluentd-gcp-memory-limit
```

Comment:
How are we going to know if the regex breaks? It's not obvious that a regex targets this anymore.